server/planning/output_formatters/ntb_event.py

```diff
@@ -50,7 +50,7 @@ def _format_doc(self, doc, item):
         priority = etree.SubElement(doc, 'priority')
         priority.text = str(item.get('priority', self.PRIORITY))
         content = etree.SubElement(doc, 'content')
-        content.text = item.get('definition_long', item.get('definition_short'))
+        content.text = item.get('definition_short', '')
```

Comment:
Because **definition_short/definition_long** can be enabled/disabled using **planning_types**, would it be better to use the following:
`content.text = item.get('definition_sort') or item.get('definition_long) or ''`

Reply:
afaik they don't want to ever export definition_long, it's kind of internal info for ntb